### PR TITLE
create initial dashboards rest api

### PR DESCRIPTION
**Pull request by tracfren**
*Description of changes:*
Initial dashboards REST API using an array for dashboard storage.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by chejimmy** [comment]
I noticed the entity definitions are missing some fields (e.g. Dashboard's description, Widget's metrics, etc). Are those to be added in the future?

**Comment by tracfren** [comment]
> I noticed the entity definitions are missing some fields (e.g. Dashboard's description, Widget's metrics, etc). Are those to be added in the future?

Yes, it's not intended to be the complete and final dashboard definition.

**Review by chejimmy — APPROVED** [review]
LGTM!

**Review by NorbertNader — APPROVED** [review]

File: apps/core/src/dashboards/dashboards.service.ts [hunk 1]
```diff
@@ -1,26 +1,110 @@
...
 
 @Injectable()
 export class DashboardsService {
-  create(createDashboardDto: CreateDashboardDto) {
-    return "This action adds a new dashboard";
+  private readonly logger = new Logger(DashboardsService.name);
+  private dashboards: Dashboard[] = [];
+
+  public async create(name: Dashboard["name"]) {
+    this.logger.log("Creating dashboard...");
```
Comment (NorbertNader) [inline apps/core/src/dashboards/dashboards.service.ts:12]: Really like this logging. So clean.

**Pull request merged**